torch/_dynamo/bytecode_transformation.py

```diff
@@ -323,15 +427,20 @@ def transform_code_object(code, transformations, safe=False):
         "co_varnames",
         "co_filename",
         "co_name",
+        "co_qualname",  # python 3.11+
```

Comment:
Also consider making a keys_3_11 list instead of inlining these, it would make the sys.version logic below much cleaner (add to list instead of remove)